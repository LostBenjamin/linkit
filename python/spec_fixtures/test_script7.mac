$var `1 + 2`
$var2 `<var> + 3`
$var3 `<var2> + 4`
<var>
<var2>
<var3>
$var4 `<var> + <var2> + <var3>`
<var4>
$var5 `<var4> * 100`
<var5>

