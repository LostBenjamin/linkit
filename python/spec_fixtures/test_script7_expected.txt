3:6:10:19:1900

